mindjunkies/home/tests/test_views.py

```
@@ -0,0 +1,88 @@
+
+    assert response.status_code == 200
+    assert "enrolled_courses" in response.context
+    assert "new_courses" in response.context
+    assert enrolled_course in response.context["enrolled_courses"]
+    assert all(course not in response.context["new_courses"] for course in response.context["enrolled_courses"])
+
+
+# ✅ Test `search_view` with a matching query
+def test_search_view_with_results(client):
+    """Test search with a query that matches course titles."""
+    course1 = baker.make(Course, title="Python Programming")
```

Ruff (0.8.2)

Local variable `course1` is assigned to but never used

Remove assignment to unused variable `course1`

(F841)